integrationTests/multiShard/transaction/interceptedResolvedBulkTx_test.go

```diff
@@ -316,10 +324,19 @@ func TestNode_InMultiShardEnvRequestTxsShouldRequireOnlyFromTheOtherShard(t *tes
 		nodes = append(nodes, tn)
 	}
 
-	var txHashesGenerated [][]byte
-	var dPool dataRetriever.PoolsHolder
-	shardCoordinator, _ := sharding.NewMultiShardCoordinator(uint32(maxShards), 0)
-	dPool, txHashesGenerated = integrationTests.CreateResolversDataPool(t, nodes, txGenerated, 0, 1, shardCoordinator)
+	senderShardId := uint32(0)
+	recvShardId := uint32(1)
+	balanceValue := big.NewInt(10000)
+	shardCoordinator, _ := sharding.NewMultiShardCoordinator(uint32(maxShards), senderShardId)
+	dPool, txHashesGenerated, txsSndAddr := integrationTests.CreateResolversDataPool(t, txGenerated, senderShardId, recvShardId, shardCoordinator)
+
+	for _, n := range nodes {
+		for _, txSenderAddr := range txsSndAddr {
+			account, _ := n.AccntState.GetAccountWithJournal(state.NewAddress(txSenderAddr))
```

Comment:
This works because we use plain address converter. Why not use a safer way to do initial minting? Might create a new minting func in integrationTests package as we might reuse it

Reply:
Done.